modules/twinkleprod.js

```diff
@@ -185,7 +185,11 @@ Twinkle.prod.callbacks = {
 
 		// Remove tags that become superfluous with this action
 		text = text.replace(/\{\{\s*([Nn]ew unreviewed article|[Uu]nreviewed|[Uu]serspace draft)\s*(\|(?:\{\{[^{}]*\}\}|[^{}])*)?\}\}\s*/g, "");
-
+		if (mw.config.get('wgNamespaceNumber') === 6) {
```

Comment:
At the top of this file the variable `namespace` is set to `article` or `file` (and eventually `book`). This is because we needed the noun itself to put in messages, so to keep it consistent I've been using `namespace === 'file'`, etc., when doing these checks. This is also helpful because you don't need to remember what namespace `6` is.